ui-grid/ui-grid.d.ts

```diff
@@ -2823,12 +2823,12 @@ declare module uiGrid {
              * returns all selected rows as gridRows
              * @returns {Array<IGridRow>} The selected rows
              */
-            getSelectedGridRows(): Array<IGridRow>;
+            getSelectedGridRows(): Array<uiGrid.IGridRow>;
             /**
              * Gets selected rows as entities
              * @returns {Array<any>} Selected row entities
              */
-            getSelectedRows(): Array<IGridRow>;
+            getSelectedRows(): Array<uiGrid.IGridRow>;
```

Comment:
This is actually wrong. See my PR #5684 waiting for @vvakame to merge. The return type should be Array<any>. You should change this so that your fix doesn't overwrite mine. :)